torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -123,6 +124,11 @@ def _evaluate_opaque_descriptor(descriptor: AOTInput, args: Sequence[FxValue]):
             # base.attr
             base_value = _evaluate_opaque_descriptor(base, args)
             return getattr(base_value, attr)
+        case SubclassMethodAOTInput(base, target, args=margs, kwargs=kwargs):
+            # base.attr(margs, kwargs)
+            base_value = _evaluate_opaque_descriptor(base, args)
+            method = getattr(base_value, target)
+            return method(*margs, **kwargs)
```

Comment:
do you mind helping me understand the use case? The main way that tensor subclasses integrate with compile today is that they desugar into their inner plain tensors, and so at runtime the only thing we ever need to do with subclasses is read their inner tensor attributes off of them (in case the user passed us a subclass input that we need to flatten at runtime).

Reply:
Right, thanks! I imagine then that even though this object can represent arbitrary method calls on subclasses graph inputs, in practice we are only going to apply this to a very small number of things (process group access on DTensor inputs). Do you mind leaving a comment / paper trail somewhere mentioning that the main use case for this is around ProcessGroup runtime handling?